### PR TITLE
Monitor your own workload - RNs 4.5

### PR DESCRIPTION
**Pull request by codyhoag**

### CODE REVIEW

**Comment by codyhoag** [comment]
@rh-max I noticed you were working on this in #22879. Can you confirm this release note is accurate?

**Comment by codyhoag** [comment]
@rh-max bump. Can you take a look at this 4.5 release note for monitoring?

**Comment by rh-max** [comment]
@openshift/openshift-team-monitoring Could you PTAL?

File: release_notes/ocp-4-5-release-notes.adoc [hunk 1]
```diff
@@ -126,6 +126,27 @@ cluster installation by sharing the node's pull secret credentials with the
 
 {product-title} {product-version} introduces the Vertical Pod Autoscaler Operator (VPA). The VPA reviews the historic and current CPU and memory resources for containers in Pods and can update the resource limits and requests based on the usage values it learns. You create individual custom resources (CR) to instruct the VPA to update all of the Pods associated with a workload object, such as a Deployment, Deployment Config, StatefulSet, Job, DaemonSet, ReplicaSet, or ReplicationController. The VPA helps you to understand the optimal CPU and memory usage for your Pods and can automatically maintain Pod resources through the Pod lifecycle.
 
+[id="ocp-4-5-cluster-monitoring"]
+=== Cluster monitoring
+
+[id="ocp-4-5-monitor-your-own-workload-tp"]
+==== Monitor your own workload (Technology Preview)
```
Comment (rh-max) [inline release_notes/ocp-4-5-release-notes.adoc:133]:
```suggestion
==== Monitor your own services (Technology Preview)
```

File: release_notes/ocp-4-5-release-notes.adoc [hunk 2]
```diff
@@ -126,6 +126,27 @@ cluster installation by sharing the node's pull secret credentials with the
...
+
+The following improvements are now available to further enhance monitoring your
+own cluster workload:
```
Comment (rh-max) [inline release_notes/ocp-4-5-release-notes.adoc:136]:
```suggestion
own services:
```

File: release_notes/ocp-4-5-release-notes.adoc [hunk 3]
```diff
@@ -126,6 +126,27 @@ cluster installation by sharing the node's pull secret credentials with the
 
 {product-title} {product-version} introduces the Vertical Pod Autoscaler Operator (VPA). The VPA reviews the historic and current CPU and memory resources for containers in Pods and can update the resource limits and requests based on the usage values it learns. You create individual custom resources (CR) to instruct the VPA to update all of the Pods associated with a workload object, such as a Deployment, Deployment Config, StatefulSet, Job, DaemonSet, ReplicaSet, or ReplicationController. The VPA helps you to understand the optimal CPU and memory usage for your Pods and can automatically maintain Pod resources through the Pod lifecycle.
 
+[id="ocp-4-5-cluster-monitoring"]
+=== Cluster monitoring
+
+[id="ocp-4-5-monitor-your-own-workload-tp"]
```
Comment (rh-max) [inline release_notes/ocp-4-5-release-notes.adoc:132]:
```suggestion
[id="ocp-4-5-monitor-your-own-services-tp"]
```

File: release_notes/ocp-4-5-release-notes.adoc [hunk 4]
```diff
@@ -126,6 +126,27 @@ cluster installation by sharing the node's pull secret credentials with the
...
+==== Monitor your own workload (Technology Preview)
+
+The following improvements are now available to further enhance monitoring your
+own cluster workload:
+
+* Allow cross-correlation of in-cluster metrics.
```
Comment (rh-max) [inline release_notes/ocp-4-5-release-notes.adoc:138]:
```suggestion
* Allow cross-correlation of the metrics of your own service with the cluster metrics.
```

File: release_notes/ocp-4-5-release-notes.adoc [hunk 5]
```diff
@@ -126,6 +126,27 @@ cluster installation by sharing the node's pull secret credentials with the
...
+* Allow user workload monitoring metrics in user namespaces in recording and
```
Comment (rh-max) [inline release_notes/ocp-4-5-release-notes.adoc:139]:
```suggestion
* Allow using metrics of services in user namespaces in recording and
```

**Review by rh-max — CHANGES_REQUESTED** [review]
Hi @codyhoag, I've made a few suggestions to make some items more precise and to call the Technology Preview in question the same way as done in the main docs.

**Pull request merged**